test/e2e/integration/frontend-test/validation.ts

```diff
@@ -120,22 +122,20 @@ describe('Validation', () => {
       .should('contain.text', texts.requiredFieldDateFrom)
       .should('contain.text', texts.next);
     cy.navPage('form').should('have.attr', 'aria-current', 'page');
-
     // Make sure all the buttons in the form are now inside errorReport, not outside of it.
     // - 4 of the button roles belong to each of the errors in the report
     // - 2 of the button roles belong to the buttons on the bottom of the form (print, next)
     cy.get(appFrontend.errorReport).findAllByRole('listitem').should('have.length', 4);
     cy.get(appFrontend.errorReport)
       .findAllByRole('button')
       .should('have.length', 4 + 2);
-
     const lastNameError = appFrontend.fieldValidation(appFrontend.changeOfName.newLastName);
     cy.get(lastNameError).should('exist').should('not.be.inViewport');
     cy.get(appFrontend.changeOfName.newLastName).should('not.be.focused');
 
     cy.get(appFrontend.errorReport)
       .get(`button:contains("${texts.requiredFieldLastName}")`)
-      .should('be.inViewport')
+      //.should('be.inViewport')
```

Comment:
Hmm..? :thinking: Did this cause the test to be flaky, or not work at all? The reason we test for this is that clicking on the submit button should scroll the page such that the submit button is in the exact same place on screen as before clicking it. Otherwise you'll click the button and the button potentially just disappears for you underneath a long list of validation errors.

Reply:
I think i commented it out to understand what was happening in the test, then i forgot to uncomment it back in.